Jenkinsfile

```diff
@@ -24,6 +24,7 @@
 mx_lib = 'lib/libmxnet.so, lib/libmxnet.a, dmlc-core/libdmlc.a, nnvm/lib/libnnvm.a'
 // mxnet cmake libraries, in cmake builds we do not produce a libnvvm static library by default.
 mx_cmake_lib = 'build/libmxnet.so, build/libmxnet.a, build/dmlc-core/libdmlc.a, build/tests/mxnet_unit_tests, build/3rdparty/openmp/runtime/src/libomp.so'
+mx_cmake_mkldnn_lib = 'build/libmxnet.so, build/libmxnet.a, build/dmlc-core/libdmlc.a, build/tests/mxnet_unit_tests, /usr/local/lib/libiomp5.so, /usr/local/lib/libmklml_gnu.so, build/3rdparty/mkldnn/install/lib/libmkldnn.so, build/3rdparty/mkldnn/install/lib/libmkldnn.so.0, /usr/local/lib/libmklml_intel.so'
```

Comment:
I don't understand why we are generating files outside the source directory. This could cause issues because I would not assume that a compile process writes files to /usr/local or any other place outside of the source directory. Since I don't have much experience with cmake, I'd like to ask @larroy and @cjolivier01 for assistance here

Reply:
@cjolivier01 /usr/local/lib/libiomp5.so /usr/local/lib/libmklml_gnu.so /usr/local/lib/libmklml_intel.so

Reply:
beats me — ask the Intel guys

Reply:
Will do. But is this an industry standard or would you expect all artefacts to be written into the source dir?

Reply:
@pengzhao-intel